hack/make-rules/test.sh

```diff
@@ -202,7 +203,7 @@ for arg; do
   fi
 done
 if [[ ${#testcases[@]} -eq 0 ]]; then
-  testcases=($(kube::test::find_dirs))
+  while IFS='' read -r line; do testcases+=("$line"); done < <(kube::test::find_dirs)
```

Comment:
fix this failure
```
In ./hack/make-rules/test.sh line 205:
  testcases=($(kube::test::find_dirs))
             ^----------------------^ SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```